docker/pypi/wmagent-mariadb/Dockerfile

```diff
@@ -0,0 +1,81 @@
+ARG TAG=11.0
+ARG MDB_TAG=$TAG
+FROM mariadb:${MDB_TAG}
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+ARG MDB_TAG=$TAG
+ENV MDB_TAG=${MDB_TAG}
+RUN echo MDB_TAG=$MDB_TAG
+
+RUN apt-get update && apt-get install -y vim less sudo wget unzip python3 pip
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps emacs-nox tcpdump && apt-get clean
+
+ENV USER=cmst1
+# ENV MDB_PORT=
+ENV UID=31961
+ENV MDB_ROOT_DIR=/data
+
+ENV MDB_BASE_DIR=$MDB_ROOT_DIR/srv/mariadb
+ENV MDB_ADMIN_DIR=$MDB_ROOT_DIR/admin/mariadb
+ENV WMA_ADMIN_DIR=$MDB_ROOT_DIR/admin/wmagent
+ENV MDB_CERTS_DIR=$MDB_ROOT_DIR/certs
+
+ENV MDB_CURRENT_DIR=$MDB_BASE_DIR/$MDB_TAG
+ENV MDB_MANAGE_DIR=$MDB_CURRENT_DIR
+ENV MDB_AUTH_DIR=$MDB_CURRENT_DIR/auth/
+ENV MDB_INSTALL_DIR=$MDB_CURRENT_DIR/install
+ENV MDB_STATE_DIR=$MDB_CURRENT_DIR/state
+ENV MDB_DATABASE_DIR=$MDB_INSTALL_DIR/database
+ENV MDB_CONFIG_DIR=$MDB_CURRENT_DIR/config
+ENV MDB_LOG_DIR=$MDB_CURRENT_DIR/logs
+ENV MDB_DEPLOY_DIR=/usr/local
+ENV MDB_ENV_FILE=$MDB_DEPLOY_DIR/deploy/env.sh
+ENV MDB_SOCKET_FILE=$MDB_CURRENT_DIR/mariadb.sock
+ENV MDB_SECRETS_FILE=$MDB_ADMIN_DIR/MariaDB.secrets
+ENV WMA_SECRETS_FILE=$WMA_ADMIN_DIR/WMAgent.secrets
+ENV WMA_DATABASE=wmagent
+
+
+RUN useradd -u $UID -m $USER
+
+# add user to sudoers file
+RUN echo "$USER ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+
+# start the setup
+RUN mkdir -p $MDB_ROOT_DIR
+
+ENV PATH="${MDB_ROOT_DIR}:${PATH}"
+
+RUN mkdir -p $MDB_CURRENT_DIR $MDB_CONFIG_DIR $MDB_MANAGE_DIR $MDB_LOG_DIR $MDB_DATABASE_DIR $MDB_STATE_DIR $MDB_AUTH_DIR
+RUN ln -s $MDB_CURRENT_DIR $MDB_BASE_DIR/current
+
+# add necessary scripts
+ADD run.sh ${MDB_ROOT_DIR}/
+ADD manage ${MDB_MANAGE_DIR}/manage
+RUN ln -s ${MDB_MANAGE_DIR}/manage ${MDB_ROOT_DIR}/manage
+
+# The $MDB_CONFIG_DIR is to be mounted from the host and my.cnf read from there
+ADD my.cnf ${MDB_CONFIG_DIR}/my.cnf
+
+ENV PATH="/opt/couchdb/bin:/usr/local/bin/:${PATH}"
```

Comment:
I suggest to combine PATH statements into one line, like this:
```
ENV PATH="/opt/couchdb/bin:/usr/local/bin/:${MDB_ROOT_DIR}:${PATH}"
```
Again, it will reduce number of steps and build time.

Reply:
ok